components/common/Header.js

```diff
@@ -1,91 +1,93 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withRouter } from 'next/router';
-import Link from 'next/link';
 import { connect } from 'react-redux';
 import classNames from 'classnames';
 
 import Clickable from 'components/common/Clickable';
-import ArticleLink from 'components/articles/ArticleLink';
+import Link from 'components/common/Link';
+import LocaleContext from 'components/common/LocaleContext';
 
 import { actions, selectors } from 'redux/ducks/auth';
 import text from 'constants/dictionary';
-import routes from 'constants/routing';
+import { Router, NAVBAR_ROUTES, ROUTES_NAMES } from 'routes';
+import { LOCALES, LANGS } from 'constants';
 
 const mapStateToProps = state => ({ user: selectors.getUser(state) });
-
 const mapDispatchToProps = { signOut: actions.signOut };
 
-/* TODO: get values for langs and links from dictionary ? */
-// const langs = [
-//   { id: 'be', value: 'беларуская' },
-//   { id: 'ru', value: 'русский' },
-//   { id: 'en', value: 'english' },
-// ];
+const checkActive = (path, name) => path.split('/').slice(-1)[0] === name;
 
-const { home, collections, partners, articles, about, login } = routes;
+const getLocaleSwithUrl = (path, lang) => {
```

Comment:
Sorry, I can't get the name of the function: what does letter ``S`` mean here ``getLocaleSwithUrl``?

Reply:
Fixed. 😄